packages/libretro/pcsx_rearmed/package.mk

```diff
@@ -43,7 +43,7 @@ make_target() {
   if [[ "$TARGET_FPU" =~ "neon" ]]; then
     make -f Makefile.libretro HAVE_NEON=1 USE_DYNAREC=1 BUILTIN_GPU=neon
   elif [ "$ARCH" == "arm" ]; then
-    make -f Makefile.libretro HAVE_NEON=0 USE_DYNAREC=1
+    make -f Makefile.libretro HAVE_NEON=0 USE_DYNAREC=1 BUILTIN_GPU=unai
```

Comment:
Which platforms does that impact? Is that only RPi0 and RPi1? We don't want unai to be the renderer on all platforms